lib/combo-renderer.coffee

```diff
@@ -101,7 +102,7 @@ module.exports =
     @lastStreak = performance.now()
     @debouncedEndStreak()
 
-    n = n * (@level + 1) if n > 0
+    n = n * (@level + 1) if n > 0 and @conf['multiplier']
```

Comment:
Fixed on [Turn the multiplier on/off added](https://github.com/JoelBesada/activate-power-mode/pull/295/commits/17877f01bd01a0dc18a304c2c03079937f14212f).